README.md

```diff
@@ -44,11 +47,20 @@ Building the API client library requires:
 
 Add this dependency to your project's POM:
 
+```xml
+<repositories>
+    <repository>
+        <id>jitpack.io</id>
+        <url>https://jitpack.io</url>
+    </repository>
+</repositories>
+```
+
 ```xml
 <dependency>
-  <groupId>com.fingerprint</groupId>
-  <artifactId>fingerprint-pro-server-api-sdk</artifactId>
-  <version>1.0.0</version>
+  <groupId>com.github.fingerprintjs</groupId>
+  <artifactId>fingerprint-pro-server-api-java-sdk</artifactId>
+  <version>v1.0.0</version>
   <scope>compile</scope>
```

Comment:
This is the default value according to the [maven documentation](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope). We can commit it as in the [jitpack example](https://jitpack.io/#fingerprintjs/fingerprint-pro-server-api-java-sdk/v1.0.0)